ui/webui/src/components/app.jsx

```diff
@@ -44,6 +44,7 @@ export const Application = () => {
     const [conf, setConf] = useState();
     const [notifications, setNotifications] = useState({});
     const [isHelpExpanded, setIsHelpExpanded] = useState(false);
+    const [helpContent, setHelpContent] = useState("");
```

Comment:
This is more a React newbie question that stroke me, you can ignore it.
Would it make sense to keep rather some reference than the element itself in the state? Is it worth effort? If so, it does not have to be part of this PR.

Reply:
Ehm, I am not sure what you mean :flushed:

Reply:
Do you mean that there should be a central data structure for storing all of the JSX objects like a hash map and that the useState should only store the selector?

Reply:
We might not be owners of the help content in some cases (because they might be provided by third-party add-ons). Also, we are not sure if the content will be always static, it could eventually contain some data generated on the spot. Based on that, I would try to keep this simple for now.